docs/components/modeler/web-modeler/process-application-versioning.md

```diff
@@ -0,0 +1,41 @@
+---
+id: process-application-versioning
+title: Process application versioning
+description: Process applications allow you to deploy multiple related files together in a single bundle.
+---
+
+Although you cannot version a process application itself, you can use [bulk milestone creation](milestones.md#bulk-milestone-creation) and version tags to save a single 'versioned' snapshot of all the process application files in one action, instead of having to create separate milestones for every file.
+
+- When you create a milestone in any process application file, a milestone is also created with the same name (version tag) for all files currently available in the process application.
+
+- This allows you to track a process application through the development lifecycle and ensure the correct version is called.
+
+## A worked example
+
+In this example, you have a process application that contains multiple files as well as the main process.
+
+- You create a new milestone for the main process and enter "1.2" as the version tag.
+- When you create this new milestone, a milestone is also created for every other file and named with the same "1.2" version tag.
+- All files in the process application now have a "version 1.2" milestone.
+
+![Diagram showing an example process application bulk milestone creation](img/process-applications/process-application-milestone-diagram.png)
+
+## Create a bulk milestone
```

Comment:
Although _technically the functionality is a bulk milestone_, it functions as a version. Versions make more sense in the context. We'll improve this in Q4

Reply:
Thanks - I have to document it as is, so technically have to say bulk milestone and so on, but as soon as we can, we can start using 'version' 👍